.eslintignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Isn't node_modules [implicitly ignored](https://eslint.org/docs/latest/use/configure/ignore#the-eslintignore-file)?